### DISCUSSION TITLE
CR6 Max with Klipper in Cura

### DISCUSSION THREAD

**Discussion by Toddi3D** [CrealityOfficial/Creality_Sonic_Pad#45] [Q&A]
Hello Guys, i looking for a nornal start and end Code for my Printer. Now i am using Klipper with Sonic Pad and my Prints stops Everytime with this Message

// {"code":"key61, "msg":"Unknown command:M201", "values": ["M201"]}
17:38:29 
// {"code":"key61, "msg":"Unknown command:M203", "values": ["M203"]}
17:38:29 
// {"code":"key61, "msg":"Unknown command:M205", "values": ["M205"]} Das wahr die Meldung

Need a new starr and end Code for Cura please

**Comment by CrealityTech** [comment]
Just dele the Mcode
